### ISSUE TITLE
Month and Year picker

### ISSUE THREAD

**Issue by AkshayNG** [jonathantribouharet/JTCalendar#275]
If I am on June 2016 and want to go on Dec 2016, then I need to swipe 6 times to achieve the same. i.e. more the gap more are the times of swipe I need to perform. It would be great to have direct picking of month and year functionality in JTCalendar. So please tell me how can I implement this direct picking functionality.

Thanks, JTCalendar !

**Comment by jonathantribouharet** [comment]
You can set a specific date directly with `[_calendarManager setDate:[NSDate date]];` but I don't think you will have an animation.

**Comment by AkshayNG** [comment]
Yes, I've implemented this method, but if I want to jump to year 2018 from say 2015, then it took more than enough efforts. So, is there any UI in JTCalendar that can let user to select month or year directly, so as to save swipe efforts ?

**Comment by jonathantribouharet** [comment]
There is nothing in the UI because I don't know how to do it. It has to be something generic.
